gpustat.py

```diff
@@ -19,8 +19,7 @@
 import locale
 import platform
 import json
-
-__version__ = '0.4.0.dev'
+import six
```

Comment:
You can simply move this import into function `jsonify`, thus the lazy import should not cause the problem in setup.py.